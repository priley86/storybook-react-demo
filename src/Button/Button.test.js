/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'
import { Button } from '../index.js'

test('Button renders properly', () => {
  const handleButtonClick = jest.fn()
  const component = renderer.create(
    <Button onClick={handleButtonClick} disabled>
      Hello Jest Snapshot! ðŸ“¸
    </Button>
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
