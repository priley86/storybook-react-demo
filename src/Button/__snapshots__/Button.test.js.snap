// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders properly 1`] = `
<button
  disabled={true}
  onClick={[Function]}
  style={undefined}
>
  Hello Jest Snapshot! ðŸ“¸
</button>
`;
